rules/CodeQuality/Rector/MethodCall/AssertTrueFalseToSpecificMethodRector.php

```diff
@@ -214,7 +215,8 @@ private function buildNewArguments(
             return [...$funcCallOrEmptyNodeArgs, ...$oldArguments];
         }
 
-        if ($funcCallOrEmptyNodeName === 'is_a') {
+        if (in_array($funcCallOrEmptyNodeName, ['is_a', 'str_contains'])) {
```

Comment:
add true to 3rd argument:

```suggestion
        if (in_array($funcCallOrEmptyNodeName, ['is_a', 'str_contains'], true)) {
```